roles/common/tasks/system-tools.yml

```diff
@@ -35,6 +34,12 @@
     - lvm2
     - libpam-cracklib
 
+- name: install specific system tools
+  apt: pkg={{ item }}
+  with_items:
+    - pstack
+  when: ansible_architecture != "ppc64le"
```

Comment:
Can we come up with a better way to install packages if available on the architecture. Otherwise can we include a comment and check against x86 rather than not PPC since these are only available on x86 architectures.